llama-index-core/llama_index/core/indices/struct_store/sql_query.py

```diff
@@ -374,6 +376,25 @@ def _get_prompt_modules(self) -> PromptMixinType:
     def sql_retriever(self) -> NLSQLRetriever:
         """Get SQL retriever."""
 
+    def _format_result_markdown(self, retrieved_nodes):
+        """Format the result in markdown."""
+        all_data = [eval(node.node.text) for node in retrieved_nodes]
```

Comment:
why do we need eval? 😅 We really shouldn't be using eval in framework code

Reply:
I made a mistake, instead of using the result attribute that came with the data ready, I was using the text, which was a string that forced me to convert the data into some structure.